config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -262,6 +262,23 @@ spec:
       {{- if .Values.csidriver.nodeSelector }}
       nodeSelector: {{- toYaml .Values.csidriver.nodeSelector | nindent 8 }}
       {{- end }}
+      affinity:
```

Comment:
could we move this to a template and reuse it for operator, webhook and csi driver? This way we'd only have to change something in one place when adding a new architecture or modifying `nodeAffinity`/`tolerations` for a different reason.

Reply:
done